akka-http-tests/src/test/scala/akka/http/scaladsl/coding/GzipSpec.scala

```diff
@@ -34,6 +34,9 @@ class GzipSpec extends CoderSpec {
     "throw an error if compressed data is just missing the trailer at the end" in {
       def brokenCompress(payload: String) = Gzip.newCompressor.compress(ByteString(payload, "UTF-8"))
 
+      println(streamEncode(ByteString("abcdefghijkl", "UTF-8")).toString())
```

Comment:
these should be removed, or turned into assertions (I guess this is debug code that remained)